api/base/filters.py

```diff
@@ -66,6 +69,33 @@ def filter_queryset(self, request, queryset, view):
             return queryset.sort(*ordering)
         return queryset
 
+    def check_serializer_fields(self, queryset, query_params, view, ordering):
+        serializer_class = view.serializer_class
+        if not queryset:
+            return []
+        sorting_params = []
+        for i, field in enumerate(query_params.getlist(self.ordering_param)):
```

Comment:
```suggestion
        for field in query_params.getlist(self.ordering_param):
```
`enumerate` doesn't seem necessary here because the index provided isn't used.